aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -100,7 +102,9 @@ public static void beforeClass() {
     crossRegionBucketName = AwsIntegTestUtil.testCrossRegionBucketName();
     accessPointName = UUID.randomUUID().toString();
     crossRegionAccessPointName = UUID.randomUUID().toString();
-    prefix = UUID.randomUUID().toString();
+
+    // adding a backslash here to cater for s3Express support.
```

Comment:
Might want to be clear here that s3 express list requests must end in a `/` unlike regular S3